{
  "info": {
    "author": "MiniExploit",
    "author_email": "miniexploitttt@gmail.com",
    "bugtrack_url": null,
    "classifiers": [
      "License :: OSI Approved :: MIT License",
      "Operating System :: OS Independent",
      "Programming Language :: Python :: 3"
    ],
    "description": "# m1n1Exception\nPython library for conveniently handling errors\n\n",
    "description_content_type": "text/markdown",
    "docs_url": null,
    "download_url": "",
    "downloads": {
      "last_day": -1,
      "last_month": -1,
      "last_week": -1
    },
    "home_page": "https://github.com/Mini-Exploit/m1n1Exception",
    "keywords": "",
    "license": "",
    "maintainer": "",
    "maintainer_email": "",
    "name": "m1n1Exception",
    "package_url": "https://pypi.org/project/m1n1Exception/",
    "platform": null,
    "project_url": "https://pypi.org/project/m1n1Exception/",
    "project_urls": {
      "Bug Tracker": "https://github.com/Mini-Exploit/m1n1Exception/issues",
      "Homepage": "https://github.com/Mini-Exploit/m1n1Exception"
    },
    "release_url": "https://pypi.org/project/m1n1Exception/2.7/",
    "requires_dist": null,
    "requires_python": ">=3.6",
    "summary": "Python library for conveniently handling errors",
    "version": "2.7",
    "yanked": false,
    "yanked_reason": null
  },
  "last_serial": 14094951,
  "urls": [
    {
      "comment_text": "",
      "digests": {
        "blake2b_256": "ac66247c1a440f6df1cd530a826355572f0bc64821f1b444e4fbc144dc5ad1a4",
        "md5": "15d0d83b1e36a934ff7114c419b8c4d3",
        "sha256": "b49103e3ea3b17a016a0b00f6b997f54d7f1cfcb9544bc3b3c9a4a760894f1c1"
      },
      "downloads": -1,
      "filename": "m1n1Exception-2.7-py3-none-any.whl",
      "has_sig": false,
      "md5_digest": "15d0d83b1e36a934ff7114c419b8c4d3",
      "packagetype": "bdist_wheel",
      "python_version": "py3",
      "requires_python": ">=3.6",
      "size": 3266,
      "upload_time": "2022-06-10T14:42:04",
      "upload_time_iso_8601": "2022-06-10T14:42:04.062048Z",
      "url": "https://files.pythonhosted.org/packages/ac/66/247c1a440f6df1cd530a826355572f0bc64821f1b444e4fbc144dc5ad1a4/m1n1Exception-2.7-py3-none-any.whl",
      "yanked": false,
      "yanked_reason": null
    },
    {
      "comment_text": "",
      "digests": {
        "blake2b_256": "598a2f85d24dce3e66f6a877c8102ac3273525060b0b7e6f95b3a18cafc8c061",
        "md5": "4408a259f3a2a82138f2c320c4bd0f0b",
        "sha256": "f7bea413aca124767a3a3a1f239034385392b5d46cdc2d34021de1ac8bc8c0df"
      },
      "downloads": -1,
      "filename": "m1n1Exception-2.7.tar.gz",
      "has_sig": false,
      "md5_digest": "4408a259f3a2a82138f2c320c4bd0f0b",
      "packagetype": "sdist",
      "python_version": "source",
      "requires_python": ">=3.6",
      "size": 2744,
      "upload_time": "2022-06-10T14:42:06",
      "upload_time_iso_8601": "2022-06-10T14:42:06.180383Z",
      "url": "https://files.pythonhosted.org/packages/59/8a/2f85d24dce3e66f6a877c8102ac3273525060b0b7e6f95b3a18cafc8c061/m1n1Exception-2.7.tar.gz",
      "yanked": false,
      "yanked_reason": null
    }
  ],
  "vulnerabilities": []
}